opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -102,8 +112,19 @@
         SpanBuilder asChildOf(Span parent);
```

Comment:
ah – I think `null` should be allowed here as a convenience... I've added comments.